dist/DISPUTE-DEBT-schema.json

```diff
@@ -0,0 +1,538 @@
+              "Nigeria",
+              "Niue",
+              "Norfolk",
+              "Norway",
+              "Oman",
+              "Pakistan",
+              "Panama",
+              "Papua New Guinea",
+              "Paraguay",
+              "Peru",
+              "Philippines",
+              "Pitcairn",
+              "Poland",
+              "Portugal",
+              "Qatar",
+              "Reunion",
+              "Romania",
+              "Russia",
+              "Rwanda",
+              "Saint Helena",
+              "Saint Kitts and Nevis",
+              "Saint Lucia",
+              "Saint Pierre and Miquelon",
+              "Saint Vincent and the Grenadines",
+              "San Marino",
+              "Sao Tome and Principe",
+              "Saudi Arabia",
+              "Senegal",
+              "Serbia",
+              "Seychelles",
+              "Sierra Leone",
+              "Singapore",
+              "Slovakia",
+              "Slovenia",
+              "Solomon Islands",
+              "Somalia",
+              "South Africa",
+              "South Georgia and the South Sandwich Islands",
+              "Spain",
+              "Sri Lanka",
+              "Sudan",
+              "Suriname",
+              "Swaziland",
+              "Sweden",
+              "Switzerland",
+              "Syrian Arab Republic",
+              "Taiwan",
+              "Tajikistan",
+              "Tanzania",
+              "Thailand",
+              "Timor-Leste",
+              "Togo",
+              "Tokelau",
+              "Tonga",
+              "Trinidad and Tobago",
+              "Tunisia",
+              "Turkey",
+              "Turkmenistan",
+              "Turks and Caicos Islands",
+              "Tuvalu",
+              "Uganda",
+              "Ukraine",
+              "United Arab Emirates",
+              "United Kingdom",
+              "Uruguay",
+              "Uzbekistan",
+              "Vanuatu",
+              "Vatican",
+              "Venezuela",
+              "Vietnam",
+              "British Virgin Islands",
+              "Wallis and Futuna",
+              "Western Sahara",
+              "Yemen",
+              "Zambia",
+              "Zimbabwe"
+            ]
+          },
+          "street": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100
+          },
+          "street2": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100
+          },
+          "street3": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100
+          },
+          "city": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 100
+          },
+          "state": {
+            "type": "string"
+          },
+          "postalCode": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "debtInformation": {
+      "type": "object",
```

Comment:
I know where were some differences between what I spec'd out and what you found as you worked this but should this be an `object`?

I have it as an array in my code. Is there a notion of an 'array' here?

```
{
  "contact_information": {
    "email": "example@place.com",
    "phone_number": "555-555-5555",
    "address_line1": "123 Main St",
    "address_line2": "Apt 1",
    "city": "Anytown"
  },
  "debt_information": [
    {
      "debt": "$227.42 Overpayment for Post-9/11 GI Bill debt for housing",
      "dispute_reason": "I don't think I owe this debt to VA",
      "support_statement": "I never had a Post-9/11 GI Bill. I'd like to have it canceled"
    },
    {
      "debt": "$300.00 for Disability Compensation and Pension debt",
      "dispute_reason": "The amount of this debt",
      "support_statement": "This is the incorrect amount, it should only be $250"
    }
  ]
}

```

Reply:
oh nice catch! you're totally right! I'll make some updates!

Reply:
Not sure, seems like we have a path forward though.